### PR TITLE
Remove/inline dependencies

### LINKED ISSUE #111: Optimize dependencies
- remove `ms`
- inline lodash/throttle, no need to include the entire package

### PR DESCRIPTION
**Pull request by quietshu**
Closes #111.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3103,7 +3098,7 @@ ms@2.0.0:
   resolved "https://registry.yarnpkg.com/ms/-/ms-2.0.0.tgz#5608aeadfc00be6c2901df5f9861788de0d597c8"
   integrity sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=
 
-ms@2.1.2, ms@^2.1.1:
+ms@^2.1.1:
```
Comment (aequasi) [inline yarn.lock]: Shouldn't these have been removed entirely?

Comment (quietshu) [reply yarn.lock]: Oh weird, I'll delete yarn.lock and regenerate one. Thanks!

Comment (quietshu) [reply yarn.lock]: @aequasi I believe is that some dependency depends on `ms`... just tried again.

Comment (aequasi) [reply yarn.lock]: Strange!

**Pull request merged**